readme.md

```diff
@@ -1722,6 +1722,7 @@ got('https://sindresorhus.com', {
 	agent: {
 		https: tunnel.httpOverHttp({
 			proxy: {
+				rejectUnauthorized: false,
```

Comment:
Not sure it's a good idea to recommend this. If we are to do it, this needs a well-written comment that explain why it's there, why it's ok to do it, and any potential downsides.

Reply:
You're right. Now I understand better the interface; don't feel it's a necessary thing!